storage/innobase/handler/ha_innodb.cc

```diff
@@ -12476,6 +12481,43 @@ innodb_change_buffering_update(
 		 *static_cast<const char*const*>(save);
 }
 
+
+#ifdef UNIV_DEBUG
+/*************************************************************//**
+Check if it is a valid value of innodb_track_changed_pages. This function is
+registered as a callback with MySQL.
+@return	0 for valid innodb_track_changed_pages */
+static
+int
+innodb_track_changed_pages_validate(
+	THD*				thd,	/*!< in: thread handle */
+	struct st_mysql_sys_var*	var,	/*!< in: pointer to system
+						variable */
+	void*				save,	/*!< out: immediate result
+						for update function */
+	struct st_mysql_value*		value)	/*!< in: incoming bool */
+{
+	long long	intbuf = 0;
+
+	if (value->val_int(value, &intbuf)) {
+		/* The value is NULL. That is invalid. */
+		return 1;
+	}
+
+	if (srv_track_changed_pages || srv_track_changed_pages_was_active) {
```

Comment:
Here or in the function header (or maybe adding a comment to srv_track_changed_pages_was_active will resolve) needed a short allowed option manipulation description comment